content/django-vs-flask.md

```diff
@@ -0,0 +1,71 @@
+# Django vs Flask: which one should you choose for your project?
+Django and Flask are among the most popular Python web frameworks.
```

Comment:
```suggestion
[Django](link_to_django_page) and [Flask](link_to_flask_page) are among the most popular Python web frameworks.
```

Reply:
I know you attached the link to the picture, but maybe the users don't try to click any image